### PR TITLE
Odis SDK get quota status

### LINKED ISSUE #9921: Create quota endpoint "wrapper" in ODIS sdk

### PR DESCRIPTION
**Pull request by isabellewei**
### Description

Add functionality in ODIS SDK to lookup quota status for an account 

Added corresponding tests

### Related issues

- Fixes #9921

### CODE REVIEW

File: packages/sdk/identity/src/odis/quota.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  sessionID?: string
+): Promise<number> {
+  const body: PnpQuotaRequest = {
+    account,
+    version: clientVersion ? clientVersion : 'unknown',
+    authenticationMethod: signer.authenticationMethod,
+    sessionID,
+  }
+
+  const response = await queryOdis<PnpQuotaResponse>(
```
Comment (alecps) [inline packages/sdk/identity/src/odis/quota.ts]: Could you also update queryOdis to take in a request generic instead? (same pattern as sendOdisDomainRequest below)

Comment (isabellewei) [reply packages/sdk/identity/src/odis/quota.ts]: updated!

Comment (alecps) [inline packages/sdk/identity/src/odis/quota.ts]: Shouldn't this take in a PnpQuotaRequest type

**Review by alecps — APPROVED** [review]
Nice work!!

**Pull request merged**